app/validations/site-validation.js

```diff
@@ -74,8 +74,14 @@ export const errorValidation = Joi.object()
       .required()
       .messages({ '*': 'Geef een geldige gemeente in' }),
     province: Joi.string()
-      .required()
-      .messages({ '*': 'Geef een geldige provincie in' }),
+      // Using.when() to conditionally require the province field
+      .when('$country', {
+        // Referencing the 'country' field at the root level
+        is: 'België', // Condition: if 'country' is 'België'
+        then: Joi.required(), // Then make 'province' required
+        otherwise: Joi.allow(null), // Optionally, allow null or undefined otherwise
```

Comment:
**Comment:** It might not be a problem here, but keep in mind that by default the empty string is not allowed. I recall we had some issues with that in OP in the past, and since then we add `.empty('')` for checks on optional strings.